pkg/antctl/raw/check/cluster/command.go

```diff
@@ -34,13 +34,15 @@ import (
 )
 
 func Command() *cobra.Command {
+	o := newOptions()
 	command := &cobra.Command{
 		Use:   "cluster",
 		Short: "Runs pre installation checks",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return Run()
+			return Run(o)
 		},
 	}
+	command.Flags().StringVar(&o.testImage, "test-image", o.testImage, "Container image override for the cluster checker")
```

Comment:
Shall we clarify that it should be the antrea-agent image?

Reply:
The default value is automatically appended to help.
```
# ./bin/antctl-linux check cluster -h
Runs pre installation checks

Usage:
  antctl-linux check cluster [flags]

Flags:
  -h, --help                help for cluster
      --test-image string   Container image override for the cluster checker (default "antrea/antrea-agent-ubuntu:v2.1.1")
```